### DISCUSSION TITLE
latexify() issue: PDE on several pages rather than one

### DISCUSSION THREAD

**Discussion by fred4ets** [pybamm-team/PyBaMM#3033] [Q&A]
Hi there,

I've just discovered latexify(), great idea!

BTW, when I want to display DFN PDE and so on for instance, all equations are on only one page, which is quite tiny and so difficult to read.

I tried it with newline kw set to false:

  model.latexify(newline=False) gives a list, yes

but this is not the case for 

  mode.latexify('DFN.pdf', newline=False)

the idea, being, behind, to do something like this:
mode.latexify('DFN1.pdf', newline=False)[0]
mode.latexify('DFN2.pdf', newline=False)[1]
mode.latexify('DFN3.pdf', newline=False)[2]....
if it is not possible to display equations on several pages.

Any suggestion?

Thanks in advance.

Regards

**Comment by brosaplanella** [comment]
The main issue with latexify is that it is quite hard to generalise. I think it might be better to explore how to allow for page breaks in equation environment in LaTeX so it prints multiple pages, rather than print each page separately manually.

**Comment by fred4ets** [comment]
Ok, thanks for the prompt answer.

I will hack the LateX code written by latexify() to do what I want.

**Reply by brosaplanella** [reply]
If you find tweaks that could be factored in, feel free to include them in #3003

**Comment by fred4ets** [comment]
Quite trivial hack, in fact.

Just change \documentclass[varwidth,12pt]{standalone} to \documentclass[12pt]{article}, that's all.

**Reply by brosaplanella** [reply]
Awesome! Feel free to add that as a comment in #3003. BTW, if you want to contribute to PyBaMM, that issue should be a nice and short first issue.
